### ISSUE TITLE
AMS >= 0.10.3 : support ruby 2.4

### ISSUE THREAD

**Issue by kofronpi** [rails-api/active_model_serializers#2004]
#### Expected behavior vs actual behavior

`active_model_serializers` should support ruby 2.4 as its release candidate is live.
the `json` gem is an indirect dependency of `ams`, and `json` supports ruby 2.4.0 only `>= 2.0.0` 

#### Steps to reproduce
Update Gemfile to ruby 2.4 and bundle install or `bundle update json`

#### Environment

ActiveModelSerializers Version `0.10.3`
Output of `ruby -e "puts RUBY_DESCRIPTION"`:  ruby 2.4.0rc1 (2016-12-12 trunk 57064) [x86_64-linux]
OS Type & Version: Ubuntu 16.04
Integrated application and version :  Rails `5.0.1.rc2`

#### Backtrace
When running `bundle update json`:
```
$ bundle update json
Fetching gem metadata from https://rubygems.org/.........
Resolving dependencies...
Bundler could not find compatible versions for gem "json":
  In Gemfile:
    json (~> 2.0.1)

    active_model_serializers (~> 0.10.3) was resolved to 0.10.3, which depends on
      jsonapi (= 0.1.1.beta2) was resolved to 0.1.1.beta2, which depends on
        json (~> 1.8)
```

#### Additonal helpful information
The `jsonapi` gem fixed this in a later beta release.

Thanks !

**Comment by beauby** [comment]
@kofronpi Sorry, I replied in the other thread (and I mistakenly suggested that not pinning `json` to post `2.0` in your Gemfile would fix your issue).

Bottom line: AMS should depend on `jsonapi` v0.1.1.beta6.

**Comment by meoooh** [comment]
@beauby Can I know that this when patch is uploaded to rubygems.org?

**Comment by bf4** [comment]
@meoooh Currently blocked by regression in https://github.com/rails-api/active_model_serializers/pull/1998 unless we make a 0.10 branch that doesn't have everything in it that's in master...

**Comment by 9mm** [comment]
Definitely interested in this, saving for later (I just tried to update my project to ruby 2.4.0, rails 5)

**Comment by dreyks** [comment]
master branch works fine for me

**Comment by grosser** [comment]
I'd love to have a release that works out of the box ... maybe just got back to the latest tag and release from a branch ?

**Comment by bf4** [comment]
I think we/I might just do that when I get back from holiday.

B mobile phone

**Comment by kofronpi** [comment]
Merged in >= 0.10.4 :yellow_heart:
